tests/test_dut_auxiliary.py

```diff
@@ -341,12 +340,14 @@ def test__run_command_exception(mocker, aux_inst, caplog):
 
 
 def test__receive_message(mocker, aux_inst):
-    response = message.Message(MESSAGE_TYPE.LOG, COMMAND_TYPE.TEST_SUITE_RUN)
+    response = message.Message(MESSAGE_TYPE.COMMAND, COMMAND_TYPE.TEST_SUITE_RUN)
```

Comment:
This confuses me... I thought the dut could only LOG and REPORT no?

Reply:
I don't know why I changed it, was changed back to LOG.